prom.go

```diff
@@ -85,14 +85,16 @@ func (c *ultraCDNCollector) Collect(ch chan<- prometheus.Metric) {
 
 				// If we can'target scrape metrics, we use the ones from cache to avoid a discontinued metric.
 				// If cache is empty, we use a 0 metric for the same reason.
+				// We multiply the local timestamp by 1000 because leaseweb's timestamps are already in milliseconds
```

Comment:
I'd move this comment to the part in the code where you actually do the multiplication.
Also, please make this a full sentence.